lib/index.js

```diff
@@ -216,6 +218,30 @@ const logsHandler = async function logsHandler(options) {
   await logs(options.function, options.tail, options.since, logStream);
 };
 
+/**
+ * Retrieve account usage stats
+ *
+ * @param {object} options - Command line options.
+ */
+const statsHandler = async function statsHandler(options) {
+  const metrics = await stats(options.since, options.until);
+  // eslint-disable-next-line no-param-reassign
+  metrics.metrics.forEach((m) => { delete m.account; });
+  const columns = ['function', 'metric', 'value', 'since', 'until'];
+  metrics.metrics = sortBy(metrics.metrics, columns);
+  if (options.json) {
+    logger.info(JSON.stringify(metrics));
+  } else {
+    if (metrics.metrics.length === 0) {
+      logger.info('No matching usage stats found');
```

Comment:
To make linter happy can be written as one `logger.info`

        logger.info(!metrics.metrics.length ? 'No matching usage stats found' : columnify(metrics.metrics, {
          showHeaders: true,
          columns,
        }));